_src/static/sass/utilities/_functions.scss

```diff
@@ -27,7 +27,7 @@
 // get line-heights from defined map values
 @function line-height($key) {
   @if map-has-key($line-heights, $key) {
-    @return map-get($line-heights, $key);
+    @return rem(map-get($line-heights, $key));
```

Comment:
Please remove this addition. We're not adding any units of measurements on purpose (see in our styleguide here - https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#line-heights).